app/api/answers/get.go

```diff
@@ -2,13 +2,41 @@ package answers
 
 import (
 	"net/http"
+	"time"
 
 	"github.com/go-chi/render"
 	"github.com/jinzhu/gorm"
 
 	"github.com/France-ioi/AlgoreaBackend/app/service"
 )
 
+// swagger:operation GET /answers/{answer_id} answers itemAnswerGet
+// ---
+// summary: Get an answer
+// description: Display an answer
+//
+//   * The user should have at least partial access rights to the users_answers.idItem item
+//
+//   * The user should be able to see answers related to his group's attempts, so
+//
+//     (a) if items.bHasAttempts = 1, then the user should be a member of the groups_attempts.idGroup team<br/>
+//     (b) if items.bHasAttempts = 0, then groups_attempts.idGroup should be equal to the user's self group
+// parameters:
+// - name: answer_id
+//   in: path
+//   type: integer
+//   required: true
+// responses:
+//   "200":
+//     "$ref": "#/responses/itemAnswerGetResponse"
+//   "400":
+//     "$ref": "#/responses/badRequestPOSTPUTPATCHResponse"
```

Comment:
Cannot be used the GET, the message is not appropriate I think. Or at least its name ;-)